.github/workflows/32bit-ci.yml

```
@@ -23,7 +23,7 @@ jobs:
         apt-get update -y
         apt-get install -y autoconf build-essential zlib1g-dev libgmp-dev curl libncurses5 libtinfo5 libncurses5-dev libtinfo-dev
         curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | BOOTSTRAP_HASKELL_NONINTERACTIVE=1 BOOTSTRAP_HASKELL_INSTALL_NO_STACK=1 sh
-    - uses: actions/checkout@v1 #This version must stay old enough to remain compatible with the container image
+    - uses: actions/checkout@v6 #This version must stay old enough to remain compatible with the container image
```

```suggestion
    - uses: actions/checkout@v1 #This version must stay old enough to remain compatible with the container image
```
As the comment says, this version should not be updated, since the newer versions don't work in the container. This can be observed in the CI failure.